examples/jsm/utils/TextureUtilsGPU.js

```diff
@@ -0,0 +1,64 @@
+import {
+	QuadMesh,
+	NodeMaterial,
+	WebGPURenderer,
+	CanvasTexture
+} from 'three';
+import { texture } from 'three/tsl';
+
+let _renderer;
+const _quadMesh = /*@__PURE__*/ new QuadMesh();
+
+export async function decompress( blitTexture, maxTextureSize = Infinity, renderer = null ) {
```

Comment:
The new `decompress()` function is now async since an internal `WebGPURenderer` has to call `init()` before performing the decompression.